pyemma/coordinates/transform/transformer.py

```diff
@@ -176,6 +176,12 @@ def describe(self):
         """ get a representation of this Transformer"""
         return self.__str__()
 
+    def dimension(self):
```

Comment:
Good idea. Please do it, I am having lunch now.

Am 12/04/15 um 14:37 schrieb Martin K. Scherer:
>
> In pyemma/coordinates/transform/transformer.py 
> <https://github.com/markovmodel/PyEMMA/pull/228#discussion_r28203055>:
>
> > @@ -176,6 +176,12 @@ def describe(self):
> >          """ get a representation of this Transformer"""
> >          return self.__str__()
> >
> > +    def dimension(self):
>
> In my opinion we should raise NotImplError here, since it might lead 
> to strange errors for dim=0 in subsequent usage of this.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/markovmodel/PyEMMA/pull/228/files#r28203055>.
>


-- 

----------------------------------------------
Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de
Mail:  Arnimallee 6, 14195 Berlin, Germany
----------------------------------------------